src/services/resource-creator.js

```diff
@@ -16,6 +16,7 @@ function ResourceCreator(model, params) {
       _.forOwn(model.associations, (association, name) => {
         if (association.associationType === 'BelongsTo') {
           promises.push(recordCreated[`set${_.upperFirst(name)}`](params[name], { save: false }));
+          recordCreated[association.identifierField] = this.params[name];
```

Comment:
handle foreign key which are also primary key

Reply:
A comment in the code would help developers that read this code in the future.